### PR TITLE
Fixed Link

### LINKED ISSUE #482: Homepage - there are 2 blue links which redirect to the same page
@caitmuenster @muffinresearch Looks like I did not catch this last week: 

STR:
1. On homepage - https://extensionworkshop.com/ in the `Need some extra help along the way?` section there's: 
"Learn more about the community" 
"Learn about enhancing the API"

Actual result:
When clicked they both redirect to https://wiki.mozilla.org/Add-ons/Community/

Expected result:
Perhaps "Learn about enhancing the API" should redirect to another page.

Notes:
Reproducible on FF69 - Win10

same URL

### PR DESCRIPTION
**Pull request by Patil2099**
Fixes #482

### CODE REVIEW

File: _includes/home/connect.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ When you build on Firefox, you build with a community of extension developers. A
 
 [Learn more about the community](https://wiki.mozilla.org/Add-ons/Community/)
 
-[Learn about enhancing the API](https://wiki.mozilla.org/Add-ons/Community/)
+[Learn about enhancing the API](https://wiki.mozilla.org/Add-ons/Community/#get-involved-in-the-community)
```
Comment (caitmuenster) [inline _includes/home/connect.md]: Hey, this should actually be linked to `/community/#get-involved-in-the-community`. This doesn't link to the wiki; it should link to https://extensionworkshop.com/community/#get-involved-in-the-community.

**Comment by Patil2099** [comment]
@caitmuenster I made the requested changes

File: _includes/home/connect.md [hunk 2]
```diff
@@ -50,7 +50,7 @@ When you build on Firefox, you build with a community of extension developers. A
 
 [Learn more about the community](https://wiki.mozilla.org/Add-ons/Community/)
```
Comment (caitmuenster) [inline _includes/home/connect.md]: Sorry I missed this earlier. This needs to link to `https://extensionworkshop.com/community` (in the code you should be able to shorten this to `/community/`.

File: _includes/home/connect.md [hunk 3]
```diff
@@ -48,9 +48,9 @@ When you build on Firefox, you build with a community of extension developers. A
 <div class="grid-container grid-x grid-padding-x align-center tiles-footer">
 <div class="cell small-12 medium-8 xlarge-6 text-center" markdown="1">
 
-[Learn more about the community](https://wiki.mozilla.org/Add-ons/Community/)
+[Learn more about the community](/Community/)
```
Comment (muffinresearch) [inline _includes/home/connect.md]: This should be lower-case `community` if it's linking to the current site.

File: _includes/home/connect.md [hunk 4]
```diff
@@ -48,9 +48,9 @@ When you build on Firefox, you build with a community of extension developers. A
...
 
-[Learn about enhancing the API](https://wiki.mozilla.org/Add-ons/Community/)
+[Learn about enhancing the API](https://extensionworkshop.com/community/#get-involved-in-the-community)
```
Comment (muffinresearch) [inline _includes/home/connect.md]: This link needs the origin (`https://extensionworkshop.com`) to be removed since it's a link to this site, that way it will work in all envs rather than just production.

**Comment by Patil2099** [comment]
@muffinresearch Is this correct now?
